src/main/java/org/spongepowered/api/CatalogType.java

```diff
@@ -50,7 +50,12 @@
      *
      * @return The unique identifier of this dummy type
      */
-    String getId();
+    @Deprecated
+    default String getId() {
+        return this.getKey().toString();
+    }
+
+    CatalogKey getKey();
```

Comment:
This method is lacking implementations, causing build failure. You should add a javadoc, too.

Reply:
I'm well aware - see the `status: wip` label.